### PR TITLE
feat(app.js): persist profile session data

### PR DESCRIPTION
**Pull request by arminsebastian**

### CODE REVIEW

File: client/src/app.js [hunk 1]
```diff
@@ -147,36 +147,49 @@
...
       return heroFactory.getOrder();
     }, function watchCallback(newVal, oldVal) {
       var stringObject = $window.JSON.stringify(newVal);
       $window.sessionStorage.setItem('heroOrder',stringObject);
-      console.log(newVal);
+      // console.log(newVal);
     }, true);
 
+    var sessionReqOrder = $window.JSON.parse($window.sessionStorage.getItem('requesterOrder'));
+    if( sessionReqOrder !== null ) {
```
Comment (darrinmn9) [inline client/src/app.js]: if (

Comment (darrinmn9) [inline client/src/app.js]: if (sessionReqOrder !== null) {

try to make this style consistent through your pull request.

File: client/src/app.js [hunk 2]
```diff
@@ -147,36 +147,49 @@
...
       return requesterFactory.getOrder();
     }, function watchCallback(newVal, oldVal) {
       var stringObject = $window.JSON.stringify(newVal);
       $window.sessionStorage.setItem('requesterOrder',stringObject);
-      console.log(newVal);
+      // console.log(newVal);
     }, true);
 
+    var sessionProfile = $window.JSON.parse($window.sessionStorage.getItem('profile'));
+    if( sessionProfile !== null ) {
```
Comment (tats-tagawa) [inline client/src/app.js]: should be `if (sessionProfile !== null) {`

**Comment by tats-tagawa** [comment]
Just two minor comments. Please fix and reopen pull request when done. Thanks.

**Pull request merged**